markdown/chapters/entombsday/playing-the-deck.md

```diff
@@ -0,0 +1,214 @@
+---
+authors: drynne
+order: 1
+title: Playing the Deck
+---
+
+## Preamble
+
+As a deck with two unique gameplans, Entombsday requires some flexibility from its pilot. Knowing the ins-and-outs of both Tin Fins and Doomsday will improve your performance with the deck, as will developing your own heuristics for when to uses which plan.
+
+Because this Wiki already contains excellent resources for how to play Turbo Doomsday, we will primarily focus on the Tin Fins plan and when to *transform* into Doomsday.
+
+## Playing Tin Fins
+
+Fundamentally, Tin Fins is an A + B reanimator combo deck.
```

Comment:
Surely you can add more than this here?

Is A+B. 
Looking to use cards like Ent to drop GrizzyB into gy, then rean with Haste with SG or GV. You can also use cards like X or Y to loop your deck or just play a slower gameplan reanimating and hitting over consecutive turns.,...

Reply:
Rewrote and expanded this section substantially